tlslite/tlsconnection.py

```diff
@@ -707,14 +754,19 @@ def _clientGetServerHello(self, settings, clientHello):
         #Get the server version.  Do this before anything else, so any
         #error alerts will use the server's version
         self.version = serverHello.server_version
+        # TODO remove when TLS 1.3 is final (server_version will be set to
+        # draft version in draft protocol implementations)
+        if self.version > (3, 4):
+            self.version = (3, 4)
 
         #Check ServerHello
         if serverHello.server_version < settings.minVersion:
             for result in self._sendError(\
                 AlertDescription.protocol_version,
                 "Too old version: %s" % str(serverHello.server_version)):
                 yield result
-        if serverHello.server_version > settings.maxVersion:
+        if serverHello.server_version > settings.maxVersion and \
+                serverHello.server_version not in settings.versions:
```

Comment:
no, because the `maxVersion` is set to logical protocol version (`(3, 4)`) while `versions` includes the actual version (the special draft version, which is like (0x7f, 23) or something)